accounts/migrations/0004_account_status.py

```diff
@@ -26,5 +26,5 @@ class Migration(migrations.Migration):
                 max_length=64,
             ),
         ),
-        migrations.RunPython(generate_status),
+        migrations.RunPython(generate_status, migrations.RunPython.noop),
```

Comment:
currently django would raise exception if we tried to reverse the migrations because the data migration has no backwards. This avoids that problem [read more](https://django.doctor/advice/C8001)